### ISSUE TITLE
Benefit titles should be marked as headings

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1356]
From Level Access:

Avoid the use of implicit headings.

[Issue]
The page contains textual content that visually appears to be a heading, but is not marked up as such.

[Impact]
Assistive technology users may have difficulty determining the page structure or efficiently navigating the page, as many assistive technology users use headings to navigate applications.

[Expected Behavior]
Ensure that all visually apparent headings are marked up as such. Developers should use heading mark up on the benefit titles to allow for more efficient navigation and more defined page structure.

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility work](https://trello.com/c/H7Q1Ycrm/93-accessibility-work)

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Screenreader Navigation](https://trello.com/c/knKtuM4T/100-accessibility-screenreader-navigation)
